client/src/components/Card/Card.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './Card.css'
+
+const Card = (props) => {
+	return(
+		<div className='card-container'>
+			<div className='card-icon'>{props.icon}</div>
+			<h4 className='card-title'>{props.title}</h4>
+			<p className='card-p'>{props.paraText}</p>
```

Comment:
Can you come up with a semantic name for this? Something other than `card-p` or `paraText`?